src/variable_tweaks.scss

```diff
@@ -0,0 +1,37 @@
+// @import url("https://fonts.googleapis.com/css?family=Nunito:400,700");
```

Comment:
Please use a more formal name for this file. variable_definition or variable_redefine, or even variables is a good enough name.